service/github/github.go

```diff
@@ -369,3 +422,7 @@ func max(x, y int32) int32 {
 	}
 	return x
 }
+
+func RevertMe() {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function RevertMe should have comment or be unexported